src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const SHORT_TERM = 3;
+  const longTerm = 7;
```

Comment:
Why don't you use a constant here.
```suggestion
  const LONG_TERM = 7;
```